spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -320,17 +352,27 @@ def main(argv=None):
                                          angle_correction=angle_correction,
                                          param_centerline=param_centerline,
                                          verbose=verbose)
+    if fname_pmj is not None:
+        mask, slices = get_slices_for_pmj_distance(fname_segmentation, fname_pmj,
+                                                   distance_pmj, extent_mask,
+                                                   param_centerline=param_centerline,
+                                                   verbose=verbose)
+
+        fname_mask_out = add_suffix(arguments.i, '_mask_csa')
+        mask.save(fname_mask_out)
```

Comment:
hum, the mask has a different orientation than the input segmentation:

**sct_image -i t2_seg.nii.gz -header**
~~~
qto_xyz:1	0.799726 0.000000 -0.020942 -19.452049 
qto_xyz:2	0.000000 0.800000 0.000000 -115.151794 
qto_xyz:3	0.020942 -0.000000 0.799726 -122.996811 
qto_xyz:4	0.000000 0.000000 0.000000 1.000000 
qform_xorient	Left-to-Right
qform_yorient	Posterior-to-Anterior
qform_zorient	Inferior-to-Superior
~~~

sct_image -i t2_seg_mask_csa.nii.gz -header
~~~
qto_xyz:1	-0.799726 0.000000 -0.020939 30.930708 
qto_xyz:2	-0.000000 0.800000 0.000000 -115.151794 
qto_xyz:3	-0.020939 -0.000000 0.799726 -121.677490 
qto_xyz:4	0.000000 0.000000 0.000000 1.000000 
qform_xorient	Right-to-Left
qform_yorient	Posterior-to-Anterior
qform_zorient	Inferior-to-Superior
~~~

Reply:
ahh yes, I can maybe put back the mask to native orientation before saving it?

Reply:
done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3429/commits/2155bf2def2d64f893165839c76efc3de4b6cde6